### ISSUE TITLE
Reality Forger Crash.

### ISSUE THREAD

**Issue by manaphoenix** [SirEdvin/UnlimitedPeripheralWorks#21]
To Reproduce, places some reality pieces and override the block to be itself.

```lua
local forger = peripheral.wrap("left") -- assuming its the reality forger block

local mimic = {
  block = "peripheralworks:reality_forger_piece" -- don't remember whats its called but you get it
}
forger.forgeReality(mimic)
```

[crash-2023-07-16_07.25.47-server.txt](https://github.com/SirEdvin/UnlimitedPeripheralWorks/files/12063878/crash-2023-07-16_07.25.47-server.txt)

**Comment by SirEdvin** [comment]
So, this is fixed in two ways:
- First, you no longer allowed to use this block as mimic target 
- Second, there is new `clearAnchors` function that allow you to cleanup created mess. Also I will add a specific recipe to cleanup statues and anchors on crafting table.

**Comment by SirEdvin** [comment]
Fixed in 1.1.0
